src/rsa.rs

```diff
@@ -432,7 +432,12 @@ enum BIGNUM {}
 
 #[cfg(feature = "rsa_signing")]
 #[allow(non_camel_case_types)]
-enum BN_BLINDING {}
+#[repr(C)]
+struct BN_BLINDING {
+    a: *mut BIGNUM,
+    ai: *mut BIGNUM,
+    counter: u32,
+}
```

Comment:
I see we already did this with `RSA` and I guess that is what allowed us to use `Box`? So, I guess I don't care strongly either way. But, we need to include the comments like this in both the Rust and C definitions of the struct:

/// Needs to be kept in sync with `struct rsa_st` (in `include/openssl/rsa.h`).